CONTRIBUTING.md

```diff
@@ -162,7 +162,7 @@ cd elasticsearch/
 ./gradlew assemble
 ```
 
-You will find the newly built packages under: `./distribution/(deb|rpm|tar|zip)/build/distributions/`.
+You will find the newly built packages under: `./distribution/(packages|archives)/(deb|rpm|tar|zip)/build/distributions/`.
```

Comment:
We probably need to be clearer here. Like separate the tar and zip with archives and rpm and deb with packages. As is, it’s ambiguous.

Reply:
Yea, you're right. I'll fix.